### PR TITLE
Updated easyRDF to version 1.1.1 so we can support php8.

### PR DESCRIPTION
**Pull request by ylebre**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
   "license": "MIT",
   "name": "pdsinterop/flysystem-rdf",
   "require": {
-    "php": "^7.3",
+    "php": "^8.0",
```
Comment (Potherca) [inline composer.json:15]: I think we might want to use `"^7.3||^8.0"` or `">=7.1.0"`, as EasyRDF supports PHP>=7.1.

No need to drop PHP7 here yet, I think?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        php: ['7.3', '7.4']
+        php: ['8.0']
```
Comment (Potherca) [inline .github/workflows/tests.yml:14]: See comment about PHP versions in `composer.json`

File: composer.json (hunk 1 above)
Comment (ylebre) [reply composer.json:15]: Agreed, I think we should align it with nextcloud 24:
```
  "require": {
    "php": ">=7.3 <=8.1"
  }
```

**Review by Potherca — APPROVED** [review]

**Review by Potherca — APPROVED** [review]

**Comment by Potherca** [comment]
Merging this as part of upgrading all packages, will look into the failing tests and follow up with a fix if needed.

**Comment by Potherca** [comment]
Broken tests resolved in 2293045e4fe5b2bcb80b8a589569b2c0f51c85d0.

**Pull request merged**